src/components/map/LocationOverview.tsx

```diff
@@ -155,19 +154,18 @@ const styles = StyleSheet.create({
   listItemContainer: {
     backgroundColor: colors.surface,
     borderRadius: normalize(12),
+    elevation: normalize(2),
     left: '4%',
     position: 'absolute',
     right: '4%',
     width: '92%',
-    // shadow:
-    elevation: 2,
     shadowColor: colors.shadowRgba,
     shadowOffset: {
-      height: 5,
+      height: normalize(5),
       width: 0
     },
-    shadowOpacity: 0.5,
-    shadowRadius: 3
+    shadowOpacity: normalize(0.5),
+    shadowRadius: normalize(3)
```

Comment:
> added normalize() throughout the files I worked in to ensure a consistent design

what was not consistent here? can you share a comparison please of what is more accurate now

Reply:
we use `normalize` throughout the app and I just added it where it wasn't used before for a consistent styling over all devices. 
we should add `normalize` every time we work on some files where it's not used but should be.